docs/cnn.md

```diff
@@ -74,23 +74,425 @@ conv = ConvModule(
 
 ### Weight initialization
 
-We wrap some initialization methods which accept a module as argument.
+> code is available at [mmcv/cnn/utils/weight_init.py](../mmcv/cnn/utils/weight_init.py)
 
-- `constant_init`
-- `xavier_init`
-- `normal_init`
-- `uniform_init`
-- `kaiming_init`
-- `caffe2_xavier_init`
-- `bias_init_with_prob`
+During training, a proper initialization strategy is beneficial to speed the
+training or obtain a higher performance. In MMCV, we provide some commonly used
+methods for initializing modules like `nn.Conv2d`. Of course, we also provide a
+high-level APIs for initializing model containing one or more
+modules.
 
-Examples:
+#### **Initialization of module**
 
-```python
-conv1 = nn.Conv2d(3, 3, 1)
-normal_init(conv1, std=0.01, bias=0)
-xavier_init(conv1, distribution='uniform')
-```
+Initializaing modules, such as `nn.Conv2d`, `nn.Linear` and so on.
```

Comment:
Initialize a `nn.Module` such as `nn.Conv2d`, `nn.Linear` in a functional way.

Reply:
got it